allauth_2fa/middleware.py

```diff
@@ -89,6 +85,9 @@ def require_2fa(self, request):
         """
         raise NotImplementedError("You must implement require_2fa.")
 
+    def is_allowed_page(self, request):
+        return request.resolver_match.url_name in self.allowed_pages
```

Comment:
Nice. I had no idea this existed. 👍

Reply:
Yeah – it's one of those [quietly documented](https://docs.djangoproject.com/en/3.2/ref/request-response/#django.http.HttpRequest.resolver_match) handy features :)